schemas/documents/seo.js

```diff
@@ -0,0 +1,38 @@
+import { MdSearch } from "react-icons/md";
```

Comment:
Please move this to `objects` directory instead of `documents`.

Reply:
Moved `seo` schema to objects folder.

Reply:
awesome. thanks